ai-gen-tools/earthquake-shakemap-urls/assets/template_interactive.html

```
@@ -0,0 +1,720 @@
+<title>__TITLE__</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)